packages/backend/package.json

```diff
@@ -23,6 +23,7 @@
     "@withorbit/api": "0.0.1",
     "@withorbit/core": "0.0.1",
     "@withorbit/firebase-support": "0.0.1",
+    "ajv": "^8.5.0",
```

Comment:
**tldr**: `ajv` is the wrong version when being used on the backend. Results in weird errors. Listing `ajv` as a dependency of the backend resolves the issue.

The situation: it looks like a dependency of another package is using `ajv` but version `6.12.6`. This becomes problematic since the `api` package is using version `8.5.0`. Thus, the file system is setup like:

```
- node_modules
  - ajv (6.12.6)
- packages
  - api
    - node_modules
      - ajv (8.5.0)
  ```

My understanding was that the backend package would use `ajv` version 8.5.0 however this is not the case. The backend is using version of 6.12.6 when being ran in the emulator.

I only noticed after reading the [migration guide for ajv](https://ajv.js.org/v6-to-v8-migration.html#potential-migration-difficulties) which marks `dataPath` as deprecated, yet this property is still appearing in our error objects when used on the backend. I only really noticed this issue after spending quite some time debugging some weird edge case that looks to be resolved in later versions of the package. 

It is worth noting that the `ajvAPIValidator.test.ts` is using the correct version of the package.

Listing `ajv` directly in the backend works, but this can't be the best solution.  I don't have too much experience with yarn monorepo's and versioning, so I have had some trouble debugging this issue further.